scanpipe/pipes/d2d.py

```diff
@@ -593,35 +613,44 @@ def match_purldb_resources(
                 f"as there are {resource_count:,d}"
             )
 
-    resource_iterator = to_resources.paginated(per_page=chunk_size)
+    resource_iterator = to_resources.iterator()
```

Comment:
We should be explicit about the `chunk_size` value.

Reply:
@tdruez I've set `chunk_size` argument on `to_resources.iterator()` to be the same as `chunk_size` from the function argument, which has a default value of 1000. Should `chunk_size` on the iterator be larger?